### PR TITLE
Refactors for better error tracking

### PR DESCRIPTION
**Pull request by elcuervo**
* Applies Rubocop styles
* Configures Rakefile
* Adds Travis

### CODE REVIEW

File: lib/imagga_auto_tag/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ImaggaAutoTag
-  VERSION = "0.2.1"
+  VERSION = '0.2.1'.freeze
```
Comment (lukechesser) [inline lib/imagga_auto_tag/version.rb:2]: Bump maybe?

**Review by lukechesser — APPROVED** [review]
Nice, looks good to me 👍

File: lib/imagga_auto_tag/version.rb (hunk 1 above)
Comment (elcuervo) [reply lib/imagga_auto_tag/version.rb:2]: Yeah. I wanted to leave the version bump to another PR so we can tag it and release it.

Comment (lukechesser) [reply lib/imagga_auto_tag/version.rb:2]: 👍 gotcha

**Comment by aaronklaassen** [comment]
Ah, nice. Yeah I've been wanting to come in and do some maintenance on this gem for a long time. :+1:

**Comment by lukechesser** [comment]
who built this garbage.

**Pull request merged**